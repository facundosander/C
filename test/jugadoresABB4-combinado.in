crearArbolJugadores

# Construyo arbol 

crearJugada
1 1 2 3 4
agregarEnPartida
crearJugador 20 20 Uruguay Juan
insertarJugadorArbolJugadores

crearJugada
1 4 3 2 1
agregarEnPartida
crearJugador 12 12 Brasil Roberto
insertarJugadorArbolJugadores

crearJugada
1 1 1 1 1
agregarEnPartida
crearJugador 30 30 Uruguay Luis
insertarJugadorArbolJugadores

crearJugada
1 3 3 3 3
agregarEnPartida
crearJugador 6 6 Uruguay Andres
insertarJugadorArbolJugadores

crearJugada
1 1 1 1 1
agregarEnPartida
crearJugador 3 3 Uruguay Emilia
insertarJugadorArbolJugadores

crearJugada
2 2 2 2 2
agregarEnPartida
crearJugador 4 4 Uruguay Ines
insertarJugadorArbolJugadores

crearJugada
1 1 1 1 1
agregarEnPartida
crearJugador 7 7 Uruguay Isabel
insertarJugadorArbolJugadores

crearJugada
1 1 1 1 1
agregarEnPartida
crearJugador 15 15 Uruguay Raul
insertarJugadorArbolJugadores

crearJugada
1 1 1 1 1
agregarEnPartida
crearJugador 25 25 Uruguay Noelia
insertarJugadorArbolJugadores

crearJugada
1 1 1 1 1
agregarEnPartida
crearJugador 22 22 Uruguay Matias
insertarJugadorArbolJugadores

crearJugada
1 1 1 1 1
agregarEnPartida
crearJugador 35 35 Uruguay Nicolas
insertarJugadorArbolJugadores

crearJugada
1 1 1 1 1
agregarEnPartida
crearJugador 33 33 Uruguay Emily
insertarJugadorArbolJugadores

crearJugada
1 1 1 1 1
agregarEnPartida
crearJugador 36 36 Uruguay Diego
insertarJugadorArbolJugadores

crearJugada
1 1 1 1 1
agregarEnPartida
crearJugador 23 23 Uruguay Martin
insertarJugadorArbolJugadores

crearJugada
1 1 1 1 1
agregarEnPartida
crearJugador 8 8 Uruguay Enrique
insertarJugadorArbolJugadores

crearJugada
1 1 1 1 1
agregarEnPartida
crearJugador 39 39 Uruguay Lucia
insertarJugadorArbolJugadores

# La maxima amplitud es 5
amplitudArbolJugadores

liberarArbolJugadores

crearArbolJugadores

# Construyo arbol 

crearJugada
1 1 1 1 1
agregarEnPartida
crearJugador 20 20 Uruguay Juan
insertarJugadorArbolJugadores

crearJugada
1 1 1 1 1
agregarEnPartida
crearJugador 12 12 Uruguay Rosa
insertarJugadorArbolJugadores

crearJugada
1 1 1 1 1
agregarEnPartida
crearJugador 30 30 Uruguay Perez
insertarJugadorArbolJugadores

crearJugada
1 1 1 1 1
agregarEnPartida
crearJugador 6 6 Uruguay Jose
insertarJugadorArbolJugadores

crearJugada
1 1 1 1 1
agregarEnPartida
crearJugador 3 3 Uruguay Sofia
insertarJugadorArbolJugadores

crearJugada
1 1 1 1 1
agregarEnPartida
crearJugador 7 7 Uruguay Emiliano
insertarJugadorArbolJugadores

crearJugada
1 1 1 1 1
agregarEnPartida
crearJugador 15 15 Uruguay Raul
insertarJugadorArbolJugadores

crearJugada
1 1 1 1 1
agregarEnPartida
crearJugador 25 25 Uruguay Noelia
insertarJugadorArbolJugadores

crearJugada
1 1 1 1 1
agregarEnPartida
crearJugador 22 22 Uruguay Matias
insertarJugadorArbolJugadores

crearJugada
1 1 1 1 1
agregarEnPartida
crearJugador 23 23 Uruguay Noelia
insertarJugadorArbolJugadores

crearJugada
1 1 1 1 1
agregarEnPartida
crearJugador 35 35 Uruguay Nicolas
insertarJugadorArbolJugadores

crearJugada
1 1 1 1 1
agregarEnPartida
crearJugador 33 33 Uruguay Emily
insertarJugadorArbolJugadores

crearJugada
1 1 1 1 1
agregarEnPartida
crearJugador 36 36 Uruguay Diego
insertarJugadorArbolJugadores

crearJugada
1 1 1 1 1
agregarEnPartida
crearJugador 26 26 Uruguay Martin
insertarJugadorArbolJugadores

crearJugada
1 1 1 1 1
agregarEnPartida
crearJugador 16 16 Uruguay Enrique
insertarJugadorArbolJugadores

crearJugada
1 1 1 1 1
agregarEnPartida
crearJugador 13 13 Uruguay Lucia
insertarJugadorArbolJugadores

# Transformo arbol en lista
serializarArbolJugadores

liberarArbolJugadores

crearPilaJugadores


# Construyo lista 

crearJugada
1 1 1 1 1
agregarEnPartida
crearJugador 36 36 Uruguay Diego

apilarPilaJugadores
liberarJugador


crearJugada
1 1 1 1 1
agregarEnPartida
crearJugador 33 33 Uruguay Emily
apilarPilaJugadores
liberarJugador

crearJugada
1 1 1 1 1
agregarEnPartida
crearJugador 26 26 Uruguay Martin
apilarPilaJugadores
liberarJugador

crearJugada
1 1 1 1 1
agregarEnPartida
crearJugador 22 22 Uruguay Matias
apilarPilaJugadores
liberarJugador

crearJugada
1 1 1 1 1
agregarEnPartida
crearJugador 16 16 Uruguay Enrique
apilarPilaJugadores
liberarJugador

crearJugada
1 1 1 1 1
agregarEnPartida
crearJugador 13 13 Uruguay Lucia
apilarPilaJugadores
liberarJugador

crearJugada
1 1 1 1 1
agregarEnPartida
crearJugador 7 7 Uruguay Emiliano
apilarPilaJugadores
liberarJugador

crearJugada
1 1 1 1 1
agregarEnPartida
crearJugador 3 3 Uruguay Sofia
apilarPilaJugadores
liberarJugador

crearJugada
1 1 1 1 1
agregarEnPartida
crearJugador 35 35 Uruguay Nicolas
apilarPilaJugadores
liberarJugador

crearJugada
1 1 1 1 1
agregarEnPartida
crearJugador 25 25 Uruguay Noelia
apilarPilaJugadores
liberarJugador

crearJugada
1 1 1 1 1
agregarEnPartida
crearJugador 15 15 Uruguay Raul
apilarPilaJugadores
liberarJugador

crearJugada
1 1 1 1 1
agregarEnPartida
crearJugador 6 6 Uruguay Jose
apilarPilaJugadores
liberarJugador

crearJugada
1 1 1 1 1
agregarEnPartida
crearJugador 30 30 Uruguay Perez
apilarPilaJugadores
liberarJugador

crearJugada
1 1 1 1 1
agregarEnPartida
crearJugador 12 12 Uruguay Rosa
apilarPilaJugadores
liberarJugador

crearJugada
1 1 1 1 1
agregarEnPartida
crearJugador 20 20 Uruguay Juan
apilarPilaJugadores
liberarJugador

# Transformo lista en arbol
deserializarArbolJugadores

Fin