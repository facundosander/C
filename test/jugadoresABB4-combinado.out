1>2>#  Construyo arbol .
3>4>5>6>7>8>9>10>11>12>13>14>15>16>17>18>19>20>21>22>23>24>25>26>27>28>29>30>31>32>33>34>35>36>37>38>39>40>41>42>43>44>45>46>47>48>49>50>51>52>53>54>55>56>57>58>59>60>61>62>63>64>65>66>67>#  La maxima amplitud es 5.
68>La amplitud del árbol es: 5.
69>70>71>#  Construyo arbol .
72>73>74>75>76>77>78>79>80>81>82>83>84>85>86>87>88>89>90>91>92>93>94>95>96>97>98>99>100>101>102>103>104>105>106>107>108>109>110>111>112>113>114>115>116>117>118>119>120>121>122>123>124>125>126>127>128>129>130>131>132>133>134>135>136>#  Transformo arbol en lista.
137>Jugador 20:  Juan, Uruguay, 20 años
Jugada: 1 
Movimientos: Derecha Derecha Derecha Derecha 
Jugador 12:  Rosa, Uruguay, 12 años
Jugada: 1 
Movimientos: Derecha Derecha Derecha Derecha 
Jugador 30:  Perez, Uruguay, 30 años
Jugada: 1 
Movimientos: Derecha Derecha Derecha Derecha 
Jugador 6:  Jose, Uruguay, 6 años
Jugada: 1 
Movimientos: Derecha Derecha Derecha Derecha 
Jugador 15:  Raul, Uruguay, 15 años
Jugada: 1 
Movimientos: Derecha Derecha Derecha Derecha 
Jugador 25:  Noelia, Uruguay, 25 años
Jugada: 1 
Movimientos: Derecha Derecha Derecha Derecha 
Jugador 35:  Nicolas, Uruguay, 35 años
Jugada: 1 
Movimientos: Derecha Derecha Derecha Derecha 
Jugador 3:  Sofia, Uruguay, 3 años
Jugada: 1 
Movimientos: Derecha Derecha Derecha Derecha 
Jugador 7:  Emiliano, Uruguay, 7 años
Jugada: 1 
Movimientos: Derecha Derecha Derecha Derecha 
Jugador 13:  Lucia, Uruguay, 13 años
Jugada: 1 
Movimientos: Derecha Derecha Derecha Derecha 
Jugador 16:  Enrique, Uruguay, 16 años
Jugada: 1 
Movimientos: Derecha Derecha Derecha Derecha 
Jugador 22:  Matias, Uruguay, 22 años
Jugada: 1 
Movimientos: Derecha Derecha Derecha Derecha 
Jugador 26:  Martin, Uruguay, 26 años
Jugada: 1 
Movimientos: Derecha Derecha Derecha Derecha 
Jugador 33:  Emily, Uruguay, 33 años
Jugada: 1 
Movimientos: Derecha Derecha Derecha Derecha 
Jugador 36:  Diego, Uruguay, 36 años
Jugada: 1 
Movimientos: Derecha Derecha Derecha Derecha 
Jugador 23:  Noelia, Uruguay, 23 años
Jugada: 1 
Movimientos: Derecha Derecha Derecha Derecha 
138>139>140>#  Construyo lista .
141>142>143>144>145>146>147>148>149>150>151>152>153>154>155>156>157>158>159>160>161>162>163>164>165>166>167>168>169>170>171>172>173>174>175>176>177>178>179>180>181>182>183>184>185>186>187>188>189>190>191>192>193>194>195>196>197>198>199>200>201>202>203>204>205>206>207>208>209>210>211>212>213>214>215>216>#  Transformo lista en arbol.
217>Jugador 3:  Sofia, Uruguay, 3 años
Jugada: 1 
Movimientos: Derecha Derecha Derecha Derecha 
Jugador 6:  Jose, Uruguay, 6 años
Jugada: 1 
Movimientos: Derecha Derecha Derecha Derecha 
Jugador 7:  Emiliano, Uruguay, 7 años
Jugada: 1 
Movimientos: Derecha Derecha Derecha Derecha 
Jugador 12:  Rosa, Uruguay, 12 años
Jugada: 1 
Movimientos: Derecha Derecha Derecha Derecha 
Jugador 13:  Lucia, Uruguay, 13 años
Jugada: 1 
Movimientos: Derecha Derecha Derecha Derecha 
Jugador 15:  Raul, Uruguay, 15 años
Jugada: 1 
Movimientos: Derecha Derecha Derecha Derecha 
Jugador 16:  Enrique, Uruguay, 16 años
Jugada: 1 
Movimientos: Derecha Derecha Derecha Derecha 
Jugador 20:  Juan, Uruguay, 20 años
Jugada: 1 
Movimientos: Derecha Derecha Derecha Derecha 
Jugador 22:  Matias, Uruguay, 22 años
Jugada: 1 
Movimientos: Derecha Derecha Derecha Derecha 
Jugador 25:  Noelia, Uruguay, 25 años
Jugada: 1 
Movimientos: Derecha Derecha Derecha Derecha 
Jugador 26:  Martin, Uruguay, 26 años
Jugada: 1 
Movimientos: Derecha Derecha Derecha Derecha 
Jugador 30:  Perez, Uruguay, 30 años
Jugada: 1 
Movimientos: Derecha Derecha Derecha Derecha 
Jugador 33:  Emily, Uruguay, 33 años
Jugada: 1 
Movimientos: Derecha Derecha Derecha Derecha 
Jugador 35:  Nicolas, Uruguay, 35 años
Jugada: 1 
Movimientos: Derecha Derecha Derecha Derecha 
Jugador 36:  Diego, Uruguay, 36 años
Jugada: 1 
Movimientos: Derecha Derecha Derecha Derecha 
218>Fin.
